docs/How-To-Create-Evergreen-Projects.md

```diff
@@ -0,0 +1,27 @@
+# How to Create Evergreen Projects
+
+This guide is meant to help users create their own projects on Evergreen without creating a ticket and waiting. 
+
+## Authorization
+
+To have access to create a project, the user must be either a project admins or super user.
+If you do not see the `New Project` button on the project settings page, or do not have access to the project settings page, you may request admin permissions from MANA or ask a current admin to add you as one.
```

Comment:
I wasn't sure on what to link because you need to specify a project to go to a project settings link but i just put "YourProject" in the link. 

I also added a note that it isnt available in legacy

And i referred the ticket creating step into the wiki as well